roles/kubernetes/node/tasks/main.yml

```diff
@@ -117,21 +117,25 @@
   tags:
     - kube-proxy
 
-- name: Modprobe nf_conntrack_ipv4
+- name: Set nf_conntrack module name
+  ansible.builtin.set_fact:
+    conntrack_module: "{{ ansible_kernel is version_compare('4.19', '>=') | ternary('nf_conntrack', 'nf_conntrack_ipv4') }}"
```

Comment:
No need to set a fact

-> That can go into either `roles/kubernetes/node/defaults/main.yml` or `roles/kubernetes/node/vars/main.yml`, depending if we want to allow the user to modify that or not. I don't think there is a reason for that (using a custom conntrack module seems pretty far-fetched), so I'd put it in vars/

Reply:
Pushed in the change. I've moved the variable to `roles/kubernetes/node/defaults/main.yml`.